setup.py

```diff
@@ -30,6 +30,7 @@
     },
 
     install_requires=[
-        'pylint>=1.4.0',
+        'pylint==1.4.4',
+        'pylint-django==0.6.1',
```

Comment:
Shouldn't these changes be reflected in `requirements.txt`? Frankly, I'm not sure why this project has both a `requirements.txt` file *and* an `install_requires` section in the setup.py (that doesn't read the `requirements.txt` file).